### PR TITLE
gtktetris-0.6.2b

### PR DESCRIPTION
**Pull request by wdlkmpx**
changes from:
https://sourceforge.net/projects/gtktetris/files/gtktetris/gtktetris-0.6.2b/

### CODE REVIEW

File: interface.c [hunk 1]
```diff
@@ -255,7 +256,7 @@ void game_start_stop(GtkMenuItem     *widget,
       from_virtual();
       move_block(0,0,0);
       current_level = options.level;
-      update_game_values(0,current_level,0);
+      update_game_values();
```
Comment (wader) [inline interface.c:259]: Oh how did this work?

**Comment by wader** [comment]
Thanks! i haven't tested but looks good

I can add you as a collaborator if you want to maintain it?

File: interface.c (hunk 1 above)
Comment (wdlkmpx) [reply interface.c:259]: The function is defined as `void update_game_values(void)`
It did compile with the extra params.. C allows crazy things.

**Comment by wdlkmpx** [comment]
The changes do work, I don't see any breakage, so this is safe to merge.

Yeah I can be a collaborator and add some changes and update a few parts.

**Comment by wader** [comment]
Sent a collaboration invite

Feel free to update the README (time for markdown?) and remove the inactive marking etc

**Comment by wdlkmpx** [comment]
Ok thanks, I'll perform some updates, currently there's a bug where the next tetromino is not shown in the small box, don't know if I can fix it, but I'm playing the game.

Collaborators cannot change project description (the inactive text) or the project topics (I suggest `gtk`, `gtk2`, `tetris`). It's something that you must do.

**Comment by wdlkmpx** [comment]
Nevermind, I forgot to tick the "Show next block" check, the app is
stable enough, it works, it just needs some polishing and gtk3
support, and a build system perhaps

**Comment by wader** [comment]
> Ok thanks, I'll perform some updates, currently there's a bug where the next tetromino is not shown in the small box, don't know if I can fix it, but I'm playing the game.

Looked thru the commits, nice work

> 
> Collaborators cannot change project description (the inactive text) or the project topics (I suggest `gtk`, `gtk2`, `tetris`). It's something that you must do.

Aha, fixed now. Let me know if you want something else changed

**Pull request merged**